scripts/pylint_extensions.py

```diff
@@ -32,13 +32,14 @@
 _PYLINT_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'pylint-1.9.4')
 sys.path.insert(0, _PYLINT_PATH)
 
+# List of punctuation allowed to used at the end of comments and docstrings.
 ALLOWED_TERMINATING_PUNCTUATIONS = ['.', '?', '}', ']', ')']
 
+# List of phrases if found inside a docstring or comment then skip
```

Comment:
If any of these phrases are found inside a docstring or comment, the punctuation and capital letter checks will be skipped for that comment or docstring.

Reply:
Done.

Reply:
Done.